demo/components/demo/components/notifications/API.md

```diff
@@ -6,6 +6,7 @@
   * `severity?: string`: The severity of the displayed message for theming.
   * `assistiveText?: string`:  The component's assistive text element.
   * `closeAssistiveText?: string = 'Close'`: The component's assistive text for the close button element.
+  * `timeout?: number`: The timeout after which, the close event will be triggered with an emitted reason of `'timeout'`.
```

Comment:
It would be better to say: `The number of milliseconds after which....`. Also, add that negative numbers/null/undefined cancel any previously set timeout.